### PR TITLE
Added footer

### PR DESCRIPTION
**Pull request by dasashreeya**
Added the footer to the main page and included some CSS for it as well.
I included the links for twitter login and link to this repo in the footer as requested.

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -1,47 +1,68 @@
-{% load static %}
+<!-- {% load static %} -->
```
Comment (kailashchoudhary11) [inline templates/base.html]:
```suggestion
{% load static %}
```

File: templates/base.html [hunk 2]
```diff
@@ -1,47 +1,68 @@
...
-  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet"
-    integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous" />
-  <link rel="stylesheet" href="{% static 'css/style.css' %}">
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" />
-  <title>{% block title %}{% endblock title %}</title>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous" />
+    <link rel="stylesheet" href="../static/css/style.css">
```
Comment (kailashchoudhary11) [inline templates/base.html]:
```suggestion
    <link rel="stylesheet" href="{% static 'css/style.css' %}">
```

**Review by kailashchoudhary11 — CHANGES_REQUESTED** [review]
Please Revert those 2 things, Everything else seems fine to me!

File: templates/base.html (hunk 2 above)
Comment (dasashreeya) [reply templates/base.html]: I had to change this because the CSS page was not loading on the website.

Comment (kailashchoudhary11) [reply templates/base.html]: > I had to change this because the CSS page was not loading on the website.

what error were you getting?

Comment (dasashreeya) [reply templates/base.html]: The CSS style sheet is not loading on the website. I am not able to link the style sheet successfully with this format.

Comment (kailashchoudhary11) [reply templates/base.html]: > The CSS style sheet is not loading on the website. I am not able to link the style sheet successfully with this format.

@dasashreeya  Can you share the screenshot of the error here or on [discord](https://discord.com/users/kai_11#4616)

**Review by kailashchoudhary11 — APPROVED** [review]
LGTM Thank you! Don't forget to star the repository! Also let me know if you would like to contribute more!

**Pull request merged**